docs/integrations/unity-catalog-kuzu.md

```diff
@@ -0,0 +1,217 @@
+
+```sql
+ATTACH [CATALOG_NAME] AS [alias] (dbtype UC_CATALOG)
+```
+
+- `CATALOG_NAME`: The catalog name to attach to in the Unity Catalog
+- `alias`: Database alias to use in Kuzu - If not provided, the catalog name will be used.
+  When attaching multiple databases, it's recommended to use aliasing.
+
+For the example shown below, we will use the following command that attaches to the `default` schema under the `unity` catalog:
+
+```sql
+ATTACH 'unity' AS unity (dbtype UC_CATALOG);
+```
+
+**Note:** Kuzu attaches to the `default` schema under the given catalog name. Specifying a custom schema to attach to is not currently supported.
+
+
+#### 2. Data type mapping from Unity Catalog to Kuzu
+
+The table below shows the mapping from Unity Catalog's type to Kuzu's type:
+| Data type in Unity Catalog         | Corresponding data type in Kuzu |
+|-----------------------------|----------------------------------|
+| BOOLEAN                     | BOOLEAN                           |
+| BYTE                        | UNSUPPORTED                          |
+| SHORT                       | INT16                                 |
+| INT                    | INT32                                 |
+| LONG                       | INT64                                 |
+| DOUBLE                     | DOUBLE                                 |
+| FLOAT                      | FLOAT                                 |
+| DATE                    | DATE                                 |
+| TIMESTAMP                    | TIMESTAMP                                 |
+| TIMESTAMP_NTZ                   | UNSUPPORTED                                 |
+| STRING                   | STRING                                 |
+| BINARY                       | UNSUPPORTED                      |
+| DECIMAL   | DECIMAL                                 |
+
+#### 3. Scan data from table
+
+Finally, we can utilize the `LOAD FROM` statement to scan the `numbers` table. Note that you need to prefix the 
+external `numbers` table with the database alias (in our example `unity`). See the `USE` statement which allows you to
+skip this prefixing for a specific default database.
+
+```sql
+LOAD FROM unity.numbers
+RETURN *
+```
+
+Result:
+
+```
+┌────────┬────────────┐
+│ as_int │ as_double  │
+│ INT32  │ DOUBLE     │
+├────────┼────────────┤
+│ 564    │ 188.755356 │
+│ 755    │ 883.610563 │
+│ 644    │ 203.439559 │
+│ 75     │ 277.880219 │
+│ 42     │ 403.857969 │
+│ 680    │ 797.691220 │
+│ 821    │ 767.799854 │
+│ 484    │ 344.003740 │
+│ 477    │ 380.678561 │
+│ 131    │ 35.443732  │
+│ 294    │ 209.322436 │
+│ 150    │ 329.197303 │
+│ 539    │ 425.661029 │
+│ 247    │ 477.742227 │
+│ 958    │ 509.371273 │
+└────────┴────────────┘
+```
+
+Currently, Kuzu only supports scanning from Delta Lake tables registered in the Unity Catalog.
+
+
+#### 4. `USE` statement
+
+You can use the `USE` statement for attached Unity Catalog to use a default Unity Catalog name (without an alias)
```

Comment:
This is not very clear. Let's do this:

If you want to refer to Delta tables without aliasing, e.g., referring to `unity.numbers`  simply as `numbers`, you can also ue the `USE` command to set a default namespace. Consider the same attached Unity Catalog as above:

(Let's remove the sentence starting with: `This can be used ...`)